.parcelrc

```diff
@@ -9,6 +9,7 @@
     "bundle-text:*.svg": ["@parcel/transformer-svg", "@parcel/transformer-inline-string"],
     "*.{md,mdx}": ["parcel-transformer-mdx-docs"],
     "*.svg": ["@parcel/transformer-svg-react"],
+    "*.global.css": ["parcel-transformer-css-global", "..."],
```

Comment:
This hack will go away once https://github.com/parcel-bundler/parcel/pull/9301 lands.